src/server.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+
+const port = 3000;
+
+app.get('/dashboard', (req, res, next) => {
+    res.send("<h1>I'm Alive!</h1>")
+});
+
+app.listen(port, () => {
+    console.log('Server is alive.');
```

Comment:
No port number specified.

```suggestion
    console.log('Server is alive.', port);
```